src/backend/machobj.c

```diff
@@ -2310,6 +2310,7 @@ void MachObj::addrel(int seg, targ_size_t offset, symbol *targsym,
         unsigned targseg, int rtype, int val)
 {
     Relocation rel;
+    memset(&rel, 0, sizeof(rel));
```

Comment:
Ah, OK, I see what's going on. I added validation to `Outbuffer` so that I can see in the stack trace when garbage is written to `Outbuffer`, not when `Outbuffer` writes said garbage to a file. This results in false positives when `Outbuffer` is not being used to buffer things later written to disk.